### DISCUSSION TITLE
How to build custom theme

### DISCUSSION THREAD

**Discussion by Wulfheart** [filamentphp/filament#9083] [Help] [answered]
Hello,

how can I build the tailwind of a custom theme.
It just does not seem to just run `npm run build`. How can I "register" the custom `tailwind.config.js`?

Thank you in advance.

**Comment by leandrocfe** [comment, marked as answer]
Did you do this? https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme

**Reply by Wulfheart** [reply]
Yes. But somehow, I can't describe how, it works now. Thank you for your answer.
